Activity 01

Droxinbox opened the pull request:
The activity number 01 is now completed and ready for its respective review.

Droxinbox committed:
Activity 01

The activity number 01 is now completed and ready for its respective review.

erobledoapex commented:
Las instrucciones también piden una captura de pantalla para cada entrega

erobledoapex commented on ExercisePart1/src/app/app.component.ts:
```diff
@@ -5,6 +5,37 @@ import { Component } from '@angular/core';
+  }
+
+  stateChangeStyle() {
+    if (this.activeStyle)
```
esto puede ser un one liner así:
this.activeStyle = !this.activeStyle;

erobledoapex commented on ExercisePart1/src/app/app.component.ts:
```diff
@@ -5,6 +5,37 @@ import { Component } from '@angular/core';
+  };
+
+  stateChangeClass() {
+    if (this.activeClass)
```
esto puede ser un one liner así:
this.activeClass = !this.activeClass;

erobledoapex commented on ExercisePart1/src/app/app.component.ts:
```diff
@@ -5,6 +5,37 @@ import { Component } from '@angular/core';
+  activeClass: boolean = false;
+  activeStyle: boolean = false;
+
+  red = {
```
le hace falta el typing a las variables red, purple y green
El typing correcto es { [nameOfAttr: string]: string }

erobledoapex commented on ExercisePart1/src/app/app.component.html:
```diff
@@ -1,336 +1,48 @@
-  .pill:hover {
-    background: color-mix(in srgb, var(--pill-accent) 15%, transparent);
-  }
+<main class="main">
```
por favor crea un componente por cada ejercicio

Droxinbox committed:
Activity 01 ver 2.0

+ Activity 01 has been converted into a component.
+ Unnecessary lines of code have been removed.

Droxinbox commented:
![imagen](https://github.com/Droxinbox/AngularPractices/assets/36482757/b9863cac-75db-457e-876f-5da4c05704e7)

Droxinbox commented:
![imagen](https://github.com/Droxinbox/AngularPractices/assets/36482757/20d9d2d5-f4fb-41b0-b779-d348f64ed294)

Droxinbox committed:
Merge branch 'main' into Activity_01

Droxinbox committed:
Update app.module.ts

Minor issues ( , )
